meson.build

```diff
@@ -0,0 +1,81 @@
+project('D-YAML', 'd',
+    meson_version: '>=0.40.0',
+    subproject_dir: 'contrib/subprojects'
+)
+
+project_version      = '0.5.4'
+project_soversion    = '0'
+
+src_dir = include_directories('source/')
+pkgc = import('pkgconfig')
+
+dyaml_src = [
+    'source/dyaml/composer.d',
+    'source/dyaml/testreader.d',
+    'source/dyaml/tag.d',
+    'source/dyaml/dumper.d',
+    'source/dyaml/emitter.d',
+    'source/dyaml/node.d',
+    'source/dyaml/testcompare.d',
+    'source/dyaml/serializer.d',
+    'source/dyaml/fastcharsearch.d',
+    'source/dyaml/unused.d',
+    'source/dyaml/testemitter.d',
+    'source/dyaml/escapes.d',
+    'source/dyaml/testinputoutput.d',
+    'source/dyaml/testtokens.d',
+    'source/dyaml/scanner.d',
+    'source/dyaml/loader.d',
+    'source/dyaml/testerrors.d',
+    'source/dyaml/representer.d',
+    'source/dyaml/reader.d',
+    'source/dyaml/event.d',
+    'source/dyaml/stream.d',
+    'source/dyaml/nogcutil.d',
+    'source/dyaml/anchor.d',
+    'source/dyaml/all.d',
```

Comment:
In that case, in case someone uses it and links with the library, they will get a linker error (something about ModuleInfo being missing I guess).
I wonder whether it makes sense to just drop the `all.d` file alltogether, as it has been deprecated for quite a while now...